frontend/src/components/smart/CharacterList.vue

```diff
@@ -142,13 +163,29 @@ export default {
       haveChangeTraitLightning: 0,
       targetTrait: '',
       currentCharacterId: null,
-      options: []
+      options: [],
+      haveCharacterCosmetic1: 0,
+      haveCharacterCosmetic2: 0,
+      haveCharacterCosmetics: [0],
+      targetSkin: '',
+      showCosmetics: true,
+      characterCosmeticsNames: [
```

Comment:
I would suggest to extract it to enum and use it here, this way it can be modified in one place if it would be used someplace else.